src/context/toast/constants.ts

```diff
@@ -0,0 +1,41 @@
+import { Variant } from "@leafygreen-ui/toast";
```

Comment:
I'm not 100% sure what this comment means so correct me if I'm wrong, but I do have a `constants.ts` and `types.ts` file in this folder. I didn't put them in the general `constants` and `types` folders because in my Parsley PR Mohamed recommended against it: https://github.com/evergreen-ci/parsley/pull/46#discussion_r980108922